SalesforceCommerceCloud/pwa-kit
# Example: using `useServerContext` together with `useQuery` hook

# How to Test-Drive This PR

```bash
npm ci # at the root of monorepo
cd packages/template-typescript-minimal
npm start
```

And then verify:
- **On the browser:** the homepage responds with 404 http status
- **On the server:** 1 log is printed - although the component is rendered twice (pre-pass and a render) on the server, `useServerContext` callback will be run only once (after the pre-pass).
